docs/skyrim_platform/new_types.md

```diff
@@ -19,3 +19,61 @@ SkyrimPlatform currently only adds one type: `TESModPlatform`. Instances of this
 - `pushWornState`, `addItemEx` - add / remove items from def. ExtraData.
 - `updateEquipment` - update equipment (unstable).
 - `resetContainer` - clear the base container.
+
+# Enums added by Skyrim Platform
+
+Skyrim Platform adds [Typescript enums][Ts-Enum] for both convenience and preventing bugs.
+
+Here's a list of enum values declared in `skyrimPlatform.ts`.
+
+- `DxScanCode`. Key mappings from [Input Script][].
+
+  ```ts
+  Input.isKeyPressed(DxScanCode.F2);
+  ```
+
+<!-- TODO: Uncomment when these enums are actually published -->
```

Comment:
The irony is that they land with the same PR. So we need to uncomment before this PR can be accepted :)

Reply:
Done.